.github/workflows/publish-account-settings.yml

```diff
@@ -0,0 +1,45 @@
+# Description: This workflow runs unit + e2e tests on `account-settings/main`.
+#
+# Triggered by: merge to `account-settings/main` branch
+
+name: Test / Account Settings
```

Comment:
This runs on every merge to `account-settings/main`. Once the feature branch is more mature, I'll set up tagged publish. For now, I'm only running e2e on it.